bin/detail/toolchain_table.py

```diff
@@ -284,6 +284,9 @@ def verify(self):
 
 if platform.system() == 'Darwin':
   toolchain_table += [
+      Toolchain('ios', 'Xcode', ios_version='11.0'), # the ios.cmake toolchain is supposed to be compatible 
```

Comment:
`ios_version` is used to read environment variable `IOS_X_Y_DEVELOPER_DIR`:
* https://github.com/ruslo/polly/blob/master/bin/detail/ios_dev_root.py

We are choosing default Xcode hence this variable should not be set.

Reply:
ok, I tried to aply what you wrote.
I cannot test it however. 
Is this ok ?